### ISSUE TITLE
MOPAC Codecov

### ISSUE THREAD

**Issue by dgasmith** [MolSSI/QCEngine#214]
MOPAC Codecov is not being uploaded due to "git not found":

```
/bin/sh: git: command not found
/bin/sh: hg: command not found
/bin/sh: git: command not found
/bin/sh: hg: command not found

      _____          _
     / ____|        | |
    | |     ___   __| | ___  ___ _____   __
    | |    / _ \ / _  |/ _ \/ __/ _ \ \ / /
    | |___| (_) | (_| |  __/ (_| (_) \ V /
     \_____\___/ \____|\___|\___\___/ \_/
                                    v2.0.15

==> Detecting CI provider
    Error running `git rev-parse --abbrev-ref HEAD || hg branch`: None
  -> Got branch from git/hg
  -> Got sha from git/hg
==> Preparing upload
Error: Commit sha is missing. Please specify via --commit=:she
```

This used to work, so unsure what happened. @Lnaden any ideas?

https://dev.azure.com/MolSSI/QCArchive/_build/results?buildId=107&view=logs&j=0e9986bc-4438-57a6-7391-1704fabd60a9&t=9bc28805-ff26-57a6-76fd-c5967bb8a1e9

**Comment by Lnaden** [comment]
No, I don't know what happened here. I also am having issues looking at the logs from earlier pipelines for some reason.

> This used to work, so unsure what happened

Are we sure? Azure did not detect this as a failure, so maybe this part has never worked? If that were the case, then I would assume `git` is not part of the docker image executing that code. Its about the only thing I can think which makes sense unless I am missing something obvious.

**Comment by dgasmith** [comment]
Yes, codecov used to show this part populated with coverage data. They could have changed what those base images were.

**Comment by Lnaden** [comment]
From the builds I can see going back to Dec 17th, it has not worked since then. My initial guess is that Azure's bash command does not provide `git` or `hg` by default anymore on their paths? Its something to test.
